### PR TITLE
Alexander Msumba- South Africa-1. JavaScript-Core-1- week-1

### PR DESCRIPTION
**Pull request by MSUMBA-art**
# Your Details

- Your Name: Alexander
- Your City: Cape Town South Africa
- Your Slack Name: Alexander Msumba

# Homework Details

- Module: JavaScript-Core-1-Week1

- Week: 1

### CODE REVIEW

**Comment by KudahShambare** [comment]
Hello, Alexander. Your pull request title should be in the City Name Module and WEEK

**Comment by MSUMBA-art** [comment]
Thanks, edited. I was rushing for time 🤣🤣🤣🤣 poor me

**Review by SerialCarKey — CHANGES_REQUESTED** [review]
Hi Alexander, clean code on most of the Mandatory, however the syntax and logical error,js is still showing small issues.

function trimWord(word) {
  return trimWord();
}
for example .

the code needs to be reviewed, you can run to test the code & see if you passed on the terminal.

If you need any assistance, feel free to contact me anytime.

**Comment by MSUMBA-art** [comment]
Thanks buddy, I will fix

**Comment by KudahShambare** [comment]
I have realised that your pull request is popping up a conflicting files notice. I tried to check it out but I couldn't identify the problem.

**Review by theTrainMan — COMMENTED** [review]
Looking great

File: exercises/I-floats/exercise.js [hunk 1]
```diff
@@ -1,2 +1,7 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+let total = numberOfStudents + numberOfMentors;
+let percentageStudents = Math.round((numberOfStudents / total) * 100);
```
Comment (he-man-david) [inline exercises/I-floats/exercise.js:4]: great job!

File: exercises/K-functions-parameters/exercise2.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 // Declare your function first
+function divide() {
```
Comment (he-man-david) [inline exercises/K-functions-parameters/exercise2.js:2]: where are the params?

File: exercises/K-functions-parameters/exercise3.js [hunk 3]
```diff
@@ -1,4 +1,8 @@
 // Write your function here
+function createGreeting(name) {
+    return "Hello my name is Daniel";
```
Comment (he-man-david) [inline exercises/K-functions-parameters/exercise3.js:3]: where is name param used?

File: mandatory/1-syntax-errors.js [hunk 4]
```diff
@@ -4,13 +4,13 @@ function addNumbers(a b c) {
...
 }
 
-function introduceMe(name, age)
+function introduceMe(name, age) {
 return "Hello, my name is " + name "and I am " age + "years old";
-
+}
 function getTotal(a, b) {
-    total = a ++ b;
+    total = a + b;
```
Comment (he-man-david) [inline mandatory/1-syntax-errors.js:11]: avoid declaring global variables like this https://stackoverflow.com/questions/10525582/why-are-global-variables-considered-bad-practice

**Pull request closed without merging**